src/main/java/com/springcore/dependencyInjection/Student.java

```diff
@@ -1,30 +1,30 @@
-//package com.springcore.dependencyInjection;
-//
-//import lombok.Getter;
-//import lombok.Setter;
-//
-//@Getter
-//@Setter
-//public class Student {
-//    private int id;
-//    private String name;
-//    private String address;
-//
-//    public Student() {
-//    }
-//
-//    public Student(int id, String name, String address) {
-//        this.id = id;
-//        this.name = name;
-//        this.address = address;
-//    }
-//
-//    @Override
-//    public String toString() {
-//        return "Student{" +
-//                "id=" + id +
-//                ", name='" + name + '\'' +
-//                ", address='" + address + '\'' +
-//                '}';
-//    }
-//}
+package com.springcore.dependencyInjection;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Student {
+    private int id;
+    private String name;
+    private String address;
+
+    public Student() {
+    }
+
+    public Student(int id, String name, String address) {
```

Comment:
add final

Reply:
the final keyword has been added to all variables which are not required to change